src/module/actor/flows/DamageApplicationFlow.ts

```diff
@@ -42,11 +42,6 @@ export class DamageApplicationFlow {
         damage = this.changeStunToPhysicalForGrunts(actor, damage);
 
         await actor.addDamage(damage);
-        
-
-        // NOTE: For stuff like healing the last wound by magic, it might also be interesting to store and give
-        //       an overview of each damage/wound applied to select from.
-        // await this.update({'data.track': this.data.data.track});
```

Comment:
Why did you removed this? I think this is still a feature to think about

Reply:
https://github.com/SR5-FoundryVTT/SR5-FoundryVTT/issues/962